recipes/cairo/all/conanfile.py

```diff
@@ -0,0 +1,253 @@
+                    args.append('FREETYPE_PATH=%s' % self.deps_cpp_info['freetype'].rootpath)
+                    args.append('GOBJECT_PATH=%s' % self.deps_cpp_info['glib'].rootpath)
+                    
+                    env_build.make(args=args)
+                    env_build.make(args=['-C', os.path.join('util', 'cairo-gobject')] + args)
+
+    def _get_env_build(self):
+        if self._env_build:
+            return self._env_build
+
+        pkg_config_path = os.path.abspath('pkgconfig')
+        pkg_config_path = tools.unix_path(pkg_config_path) if self.settings.os == 'Windows' else pkg_config_path
+
+        configure_args = ['--enable-ft' if self.options.with_freetype else '--disable-ft']
+        if self.settings.os != "Windows":
+            configure_args.append('--enable-fc' if self.options.with_fontconfig else '--disable-fc')
+        if self.settings.os == 'Linux':
+            configure_args.append('--enable-xlib' if self.options.with_xlib else '--disable-xlib')
+            configure_args.append('--enable-xlib_xrender' if self.options.with_xlib_xrender else '--disable-xlib_xrender')
+            configure_args.append('--enable-xcb' if self.options.with_xcb else '--disable-xcb')
+        configure_args.append('--enable-gobject' if self.options.with_glib else '--disable-gobject')
+        if self.options.shared:
+            configure_args.extend(['--disable-static', '--enable-shared'])
+        else:
+            configure_args.extend(['--enable-static', '--disable-shared'])
+
+        self._env_build = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        if self.settings.os == 'Macos':
+            self._env_build.link_flags.extend(['-framework CoreGraphics',
+                                            '-framework CoreFoundation'])
+        if str(self.settings.compiler) in ['gcc', 'clang', 'apple-clang']:
+            self._env_build.flags.append('-Wno-enum-conversion')
+        configure_args.append("--datarootdir=%s" % os.path.join(self.package_folder, "res"))
+
+        env_vars = {
+            "PKG_CONFIG_PATH" : pkg_config_path,
+             "NOCONFIGURE": "1"
+             }
+        with tools.environment_append(env_vars):
+            self.run("./autogen.sh", win_bash=tools.os_info.is_windows, run_environment=True)
+        self._env_build.configure(args=configure_args, pkg_config_paths=[pkg_config_path])
+        return self._env_build
+
+    def _build_configure(self):
+        with tools.chdir(self._source_subfolder):
+            # disable build of test suite
+            tools.replace_in_file(os.path.join('test', 'Makefile.am'), 'noinst_PROGRAMS = cairo-test-suite$(EXEEXT)',
+                                  '')
+            os.makedirs('pkgconfig')
+            for pc_name in  glob.glob('%s/*.pc' % self.build_folder):
+                shutil.copy(pc_name, os.path.join('pkgconfig', os.path.basename(pc_name)))
```

Comment:
Can't you avoid this copying by settings `PKG_CONFIG_PATH` to `self.build_folder`?